src/main/scala/tech/cryptonomic/conseil/util/DatabaseUtil.scala

```diff
@@ -186,7 +186,7 @@ object DatabaseUtil {
     /** maps operation type to SQL operation */
     private def mapOperationToSQL(operation: OperationType, inverse: Boolean, vals: List[String]): SQLActionBuilder = {
       val op = operation match {
-        case OperationType.between => sql"BETWEEN #${vals.head} AND #${vals(1)}"
+        case OperationType.between => sql"BETWEEN '#${vals.head}' AND '#${vals(1)}'"
```

Comment:
Will this behave as expected when comparing other columns types?

Will it not actually compare number lexicographically?

Reply:
I tested it with `<` and `>` and it works correctly.

Reply:
What about writing a test for between with a numeric type, just to be sure?

Sorry but I don't follow you when you say that you tested on `<`, `>`

Reply:
I've just added test for your case 😉